src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,95 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+@Entity
+@Table(name = "MovieSessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToOne
+    @JoinColumn(name = "movie_id")
+    private Movie movie;
+    @OneToOne
+    @JoinColumn(name = "cinema_hall_id")
+    private CinemaHall cinemaHall;
+    private LocalDateTime showTime;
+
+    public Long getId() {
+        return id;
+    }
+
+    public MovieSession setId(Long id) {
+        this.id = id;
+        return this;
+    }
```

Comment:
The `setId` method might not be necessary since the ID is auto-generated. Consider removing this setter to prevent accidental changes to the ID.